tests/conform-yup.spec.ts

```diff
@@ -1,41 +1,65 @@
 import { describe, test, expect } from 'vitest';
 import { parseWithYup, getYupConstraint } from '@conform-to/yup';
 import * as yup from 'yup';
-import { STATE } from '@conform-to/dom';
+import { type Constraint, STATE } from '@conform-to/dom';
 import { createFormData } from './helpers';
 
+// Adding custom method tuple until Yup is updated to v1 where the tuple method is built-in
+// ref: https://github.com/jquense/yup/issues/528#issuecomment-916885944
+yup.addMethod(yup.array, 'tuple', function (schema) {
+	if (!this.isType(schema)) yup.ValidationError();
+	return yup.object({
+		...Object.fromEntries(Object.entries(schema)),
+	});
+});
+
```

Comment:
It might be better to ignore tuple if it is not built-in since we can't ensure that the user has the same implementation / understanding of `tuple`.